From 2815b5d10d61d752835d32f4a691aafded4a3b75 Mon Sep 17 00:00:00 2001
Message-Id: <2815b5d10d61d752835d32f4a691aafded4a3b75.1619531915.git.wallinux@gmail.com>
From: Anders Wallin <anders.wallin@windriver.com>
Date: Mon, 9 Feb 2015 12:50:27 +0100
Subject: [PATCH] lttng ust Remove weak attribute on hidden symbols

Signed-off-by: Anders Wallin <anders.wallin@windriver.com>
---
 include/lttng/tracepoint.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/lttng/tracepoint.h b/include/lttng/tracepoint.h
index c59a96c9..36dde1e2 100644
--- a/include/lttng/tracepoint.h
+++ b/include/lttng/tracepoint.h
@@ -461,9 +461,9 @@ lttng_ust__tracepoints__destroy(void)
  * (or for the whole main program).
  */
 extern struct lttng_ust_tracepoint * const __start_lttng_ust_tracepoints_ptrs[]
-	__attribute__((weak, visibility("hidden")));
+	__attribute__((visibility("hidden")));
 extern struct lttng_ust_tracepoint * const __stop_lttng_ust_tracepoints_ptrs[]
-	__attribute__((weak, visibility("hidden")));
+	__attribute__((visibility("hidden")));
 
 /*
  * When LTTNG_UST_TRACEPOINT_PROBE_DYNAMIC_LINKAGE is defined, we do not emit a
-- 
2.31.1

