From edc29af08f515b45a2d71a0e2d7293aa37f7694c Mon Sep 17 00:00:00 2001
From: Catalin Negoita <catalin.negoita@windriver.com>
Date: Fri, 24 Jun 2016 16:30:35 +0800
Subject: [PATCH] Revert "Build: look for python >= 3.0 when building python
 bindings"

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e761d0b..dd58615 100644
--- a/configure.ac
+++ b/configure.ac
@@ -490,7 +490,7 @@ AM_CONDITIONAL([PYTHON_BINDING], [test "x$enable_python_binding" = xyes])
 if test "x$enable_python_binding" = xyes; then
   AX_PKG_SWIG($swig_version, [], [ AC_MSG_ERROR([SWIG $swig_version or newer is needed]) ])
   AS_IF([test x$enable_shared = xno], [ AC_MSG_ERROR([Python bindings require shared libraries.]) ])
-  AM_PATH_PYTHON([3.0])
+  AM_PATH_PYTHON
 
   AC_ARG_VAR([PYTHON_INCLUDE], [Include flags for python, bypassing python-config])
   AC_ARG_VAR([PYTHON_CONFIG], [Path to python-config])
-- 
1.9.1

