From 39ddf52b9d741f84d043ba7e9800e43901434540 Mon Sep 17 00:00:00 2001
Message-Id: <39ddf52b9d741f84d043ba7e9800e43901434540.1620658402.git.wallinux@gmail.com>
From: Anders Wallin <wallinux@gmail.com>
Date: Mon, 10 May 2021 16:41:09 +0200
Subject: [PATCH] test: remove failing cross-compile test

Signed-off-by: Anders Wallin <wallinux@gmail.com>
---
 tests/root_regression | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/root_regression b/tests/root_regression
index 894e8dcbf..2c9128d0c 100644
--- a/tests/root_regression
+++ b/tests/root_regression
@@ -7,7 +7,6 @@ regression/kernel/test_rotation_destroy_flush
 regression/kernel/test_select_poll_epoll
 regression/kernel/test_lttng_logger
 regression/kernel/test_callstack
-regression/kernel/test_userspace_probe
 regression/kernel/test_ns_contexts
 regression/kernel/test_ns_contexts_change
 regression/tools/live/test_kernel
-- 
2.25.1

