From 2c1b8cbce4045ec2d05039fe61c40f90b01c1125 Mon Sep 17 00:00:00 2001
From: Stanislav Vovk <stanislav.vovk@ericsson.com>
Date: Thu, 5 Apr 2018 17:25:45 +0200
Subject: [PATCH] Fix '--quiet' option functionality

Signed-off-by: Stanislav Vovk <stanislav.vovk@ericsson.com>
---
 src/bin/lttng-sessiond/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bin/lttng-sessiond/main.c b/src/bin/lttng-sessiond/main.c
index ea28fee..7fceb14 100644
--- a/src/bin/lttng-sessiond/main.c
+++ b/src/bin/lttng-sessiond/main.c
@@ -4719,7 +4719,7 @@ static int set_option(int opt, const char *arg, const char *optname)
 	} else if (string_match(optname, "no-kernel")) {
 		config.no_kernel = true;
 	} else if (string_match(optname, "quiet") || opt == 'q') {
-		lttng_opt_quiet = true;
+		config.quiet = true;
 	} else if (string_match(optname, "verbose") || opt == 'v') {
 		/* Verbose level can increase using multiple -v */
 		if (arg) {
-- 
1.7.12.4

